src/components/CharacterInfo.vue

```diff
@@ -11,8 +11,25 @@ import CharacterInfoPortrait from "./CharacterInfoPortrait.vue";
  */
 export default {
   components: [CharacterInfoInput, CharacterInfoSelect, CharacterInfoPortrait],
-  props: ["bio", "generations"],
-  emits: ["bioChange"],
+  props: ["bio", "predatorDefinitions"],
+  emits: ["bioChange", "bioChangeList"],
+  methods: {
+    /**
+     * @param {String} newPredator is value of new predator type
+     * we generate list of allowed disciplines based on it
+     * we will check if current selected predator discipline is allowed and if not, we'll reset it
+     * also we need to update list of allowed disciplines 
+     */
+    checkPredatorDiscipline(newPredator) {
+      var allowedDisciplines = this.predatorDefinitions.filter(
+        (predator) => predator.id == newPredator
+      )[0].disciplines
+      if (! allowedDisciplines.includes(this.bio.predatorDiscipline.value)) {
```

Comment:
not je unarna operacia, je take divne ze je tam medzera
ja by som cakal ze `!<vyraz>` je syntakticky cukor za `not(<vyraz>)`